lib/faraday/logging/formatter.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require 'pp'
```

Comment:
this removal is causing `NoMethodError`s because `pretty_inspect` is now undefined on hashes after updating to faraday 2.8.0

Reply:
Turns out this entire thing already happened a year ago for 2.7.0 as well, and had to be fixed by adding the require: https://github.com/lostisland/faraday/issues/1462.

Reply:
my bad!

Reply:
There I go for listening to tools and NOT adding context-preserving comments. Will add back the require and a comment.

Reply:
Now released as a patch version. Thank you so much!

https://github.com/lostisland/faraday/releases/tag/v2.8.1